libs/api/Verifier.cpp

```diff
@@ -474,7 +475,7 @@ load_byte_code(
         clean_up_ebpf_maps(maps);
     }
 
-    return result;
+    EBPF_RETURN_RESULT(result);
```

Comment:
Should there be a EBPF_LOG_ENTRY() before here?

Reply:
Added.